aries-site/src/pages/templates/wizard.mdx

```diff
@@ -73,6 +73,24 @@ If guidance can be provided in a short description, use a single column wizard.
 - On desktop, place the guidance to the right of form inputs.
 - On mobile, place the guidance above form inputs.
 
+## Alignment
+
+### Previous step button
+
+A button that enables the user to navigate to the previous step should be placed in the wizard header and aligned to the far-left side of the header.
+
+### Cancel button
+
+A button that enables the user to cancel out of the wizard should be placed in the wizard header and aligned to the far-right side of the header.
+
+### Title
+
+The title should be centered in the wizard header.
+
+### Next step button
+
+A primary action button that enables the user to advance to the next step should be placed in the footer of the wizard step and aligned to the far-right side of the footer.
```

Comment:
a primary CTA button?

Reply:
CTA button refers to the one with HPE element, in this case it's just primary action.

Reply:
gotcha thanks